MobModule/src/main/java/eu/cloudservice/cloudmob/MobModule.java

```diff
@@ -0,0 +1,75 @@
+package eu.cloudservice.cloudmob;
+
+import de.dytanic.cloudnet.event.IEventListener;
+import de.dytanic.cloudnet.lib.network.protocol.packet.PacketRC;
+import de.dytanic.cloudnet.lib.server.ServerGroupMode;
+import de.dytanic.cloudnetcore.api.CoreModule;
+import de.dytanic.cloudnetcore.api.event.network.ChannelInitEvent;
+import de.dytanic.cloudnetcore.api.event.network.UpdateAllEvent;
+import de.dytanic.cloudnetcore.network.components.MinecraftServer;
+import eu.cloudservice.cloudmob.in.PacketInAddMob;
+import eu.cloudservice.cloudmob.in.PacketInRemoveMob;
+import eu.cloudservice.cloudmob.out.PacketOutMobSelector;
+
+public class MobModule extends CoreModule implements IEventListener<UpdateAllEvent> {
```

Comment:
Having this class implement an EventListener seems like an architectural mistake